pallets/teerex/src/lib.rs

```diff
@@ -355,8 +376,18 @@ pub mod pallet {
 			mr_enclave: MREnclave,
 		) -> DispatchResultWithPostInfo {
 			T::EnclaveAdminOrigin::ensure_origin(origin)?;
+			ensure!(
+				!ScheduledEnclaveIndex::<T>::contains_key(mr_enclave),
+				Error::<T>::ScheduledEnclaveAlreadyExist
+			);
```

Comment:
I also think of such before, but in some corner cases we could (theoretically) have duplicate MRENCLAVEs, like if a newer enclave has problems and we want to revert to the old one: MR1 => MR2 => MR1, with different sidechain block numbers. It's a rare case but possible

Reply:
sure, I'll revert it.